bundles/framework/personaldata/instance.js

```diff
@@ -32,6 +32,17 @@ Oskari.clazz.define("Oskari.mapframework.bundle.personaldata.PersonalDataBundleI
         "getName": function () {
             return this.__name;
         },
+        
+        openProfileTab: function() {
+          Oskari.getSandbox().postRequestByName('userinterface.UpdateExtensionRequest', [this, 'attach']);
+          var flyout = this.plugins['Oskari.userinterface.Flyout'];
+          flyout.tabsContainer.panels.forEach(function(panel){
+            if(panel.title === "My Account") {
```

Comment:
This only works when the language is english and if the localization is not changed ever without updating. Find the correct panel some other way.